E/Exercism/Compat.toml

```diff
@@ -1,3 +1,3 @@
-[0]
+["0-0.1.2"]
 JSON = "0.19-0"
 julia = "0.7-1"
```

Comment:
Please keep these constraints.

Reply:
Registrator must've created them, I haven't specified any. How do I set them?

Reply:
No, this version 0.1.3 was excluded here. See https://julialang.github.io/Pkg.jl/v1/compatibility/

Reply:
Just to clarify, that means I **must** add compatibility version ranges for all dependencies?

Currently the Project.toml doesn't have a compat section and it never did, which is why I'm confused why the exclusion happened.

v0.1.3 Project.toml:
``` 
name = "Exercism"
uuid = "b06d6668-ed87-5b0c-b882-855c8dde7d29"
authors = ["Sascha Mann <git@mail.saschamann.eu>"]
version = "0.1.3"

[deps]
JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

v0.1.2 Project.toml:
```
name = "Exercism"
uuid = "b06d6668-ed87-5b0c-b882-855c8dde7d29"
authors = ["Sascha Mann <git@mail.saschamann.eu>"]
version = "0.1.2"

[deps]
JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Reply:
I am guessing the last release was made to METADATA, then the REQUIRE file was used, and you have not propagated those bounds to your Project file

Reply:
That makes sense. Thanks for the explanation!